src/nodes/config-client.js

```diff
@@ -4,6 +4,12 @@ module.exports = function(RED) {
 
     var VictronClient = require('../services/victronclient.js');
 
+    function serialize(key, value) {
```

Comment:
maybe `setReplacer` (per JSON.stringify documentation, there it is called `replacer`), as it is not really serializing

Reply:
pure function, so move to top level scope?

Reply:
Good point, agreed.